src/print.jl

```diff
@@ -396,7 +396,7 @@ exprstr(n::Norm) = norm_str(REPLMode, n)
 #------------------------------------------------------------------------
 ## JuMPContainer{Variable}
 #------------------------------------------------------------------------
-Base.show(io::IO, j::Union{JuMPContainer{Variable}, Array{Variable}}) = print(io, cont_str(REPLMode,j))
```

Comment:
This issue was very weird, I don't understand how this was not causing a stack overflow in 0.6 as it does in 0.7.  Will revisit after looking at 0.6 tests.